packages/varnish/src/header/Header.tsx

```diff
@@ -0,0 +1,223 @@
+                    ) : (
+                        <AppTagline>&mdash;</AppTagline>
+                    )
+                ) : null}
+                {props.tagline}
+            </LogoGrid>
+        </ConditionalAnchorWrapper>
+    );
+};
+
+interface ColumnsProps {
+    columns?: string;
+}
+
+const MaxWidthDiv = styled.div`
+    width: 100%;
+`;
+
+const StyledAppBar = styled(AppBar)`
+    background-color: white;
+    color: ${varnishColor.N9.toString()};
+    size: 22pt;
+    top: 0;
+    z-index: 950;
+    width: 100%;
+    box-shadow: 0px 4px 16px rgba(10, 41, 57, 0.08);
+    transition: top 200ms ease-in-out;
+`;
+
+interface ScrollProps {
+    children: React.ReactElement;
+    triggerTarget?: HTMLDivElement;
+}
+
+function HideOnScroll(props: ScrollProps) {
+    const { children, triggerTarget } = props;
+    const trigger = triggerTarget ? useScrollTrigger({ target: triggerTarget }) : useScrollTrigger();
+
+    return (
+        <Collapse appear={false} unmountOnExit in={!trigger}>
+            {children}
+        </Collapse>
+    );
+}
+
+interface HeaderProps {
+    children?: React.ReactNode | React.ReactNodeArray;
+    bannerAlwaysVisible?: boolean;
+    customBanner?: React.ReactNode;
+
+    // Note: Only modify this if you want the smart AI2 banner to be hidden on scroll of 
+    // a container OTHER than the main window
+    scrollTriggerTarget?: HTMLDivElement;
+}
+
+function HeaderComponent({ children, customBanner, bannerAlwaysVisible, scrollTriggerTarget }: HeaderProps) {
+    const ai2Banner = customBanner || <AI2Banner />;
+
+    return (
+        <StyledAppBar position={"sticky"}>
+            {bannerAlwaysVisible ?
+                <div>
+                    {ai2Banner}
+                </div>
+                :
+                <HideOnScroll triggerTarget={scrollTriggerTarget}>
+                    <div>
+                        {ai2Banner}
+                    </div>
+                </HideOnScroll>
+            }
+            {children ? 
+            <Toolbar>
+                <MaxWidthDiv>{children}</MaxWidthDiv>
+            </Toolbar> : null}
+        </StyledAppBar>
+    );
+}
+
+const Columns = styled.div<ColumnsProps>`
+    display: grid;
+    grid-template-columns: ${({ columns }) => columns};
+    grid-column-gap: 16px;
+    grid-row-gap: 16px;
+    width: 100%;
+    align-items: center;
+    padding: 12px 0;
+`;
+
+const MenuColumn = styled.div`
+    &&&& {
+        justify-self: right;
+        line-height: 77px;
+        a:hover {
+            text-decoration: none;
+        }
+        ul {
+            margin: -12px 0;
+        }
+    }
+`;
+
+/**
+ * Merge in the components that are tightly coupled to the <Header>,
+ * so that they can be used like:
+ *  <Header>
+ *      <Header.Columns>
+ *          ...
+ *      </Header.Columns>
+ *  </Header>
+ */
+export const Header = Object.assign(HeaderComponent, {
+    AppName,
+    AppTagline,
+    Columns,
+    // Content,
```

Comment:
I'd love to discuss this as a design decision in detail as well as a part of these header changes -- I don't know if it makes sense to add in Content as a child to the header, since these should actually be separate components? Will sync with you on this in addition to other design changes @jonborchardt

Reply:
yes, remove content
ill add it back with the context work